### DISCUSSION TITLE
Hubs Cloud Stack Failed on AWS - ExternalZoneSSLCertEast Certificate not issued in time

### DISCUSSION THREAD

**Discussion by grciprian** [mozilla/hubs#3189] [Hubs Cloud Questions] [answered]
Hi! I want to create my own stack in AWS. I had a domain berrystudio.co.uk bought on AWS Route53 that I wanted to use for the domain and internal domain but I found out the stack has some problems using a .co.uk domain. So not long ago I bought a fresh domain berrystudiointernal.com on AWS Route53 that I want to use for domain and internal domain. I already had a short domain name bshc.link bought on Route53, too. I tried to create the stack but it ended up rolling back as it got an error about ExternalZoneSSLCertEast - Certificate not issued in time. I'm attaching a screenshot for reference. Any help would be much appreciated! Thank you.

![image](https://user-images.githubusercontent.com/13333781/96388798-fe685280-11b3-11eb-96e8-d569807a197b.png)

**Comment by robinkwilson** [comment, marked as answer]
Follow **Solution 3 Certificate limit reached, must ask AWS for increase** : https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#externalzonesslcertlocalifeast-error-or-internalzonesslcert-error-or-timeout

If you want to use the .co.uk domain, the current workaround is following Domain Recipe 3 with SSL certificates. You can still do this with a Route 53 domain for a subdomain. https://hubs.mozilla.com/docs/hubs-cloud-aws-domain-recipes.html#recipe-3-domain-can-not-be-on-route-53
